src/js/view/view.js

```diff
@@ -877,16 +880,16 @@ function View(_api, _model) {
 
             _resizeOnFloat = false;
         } else if (isVisible) {
-            _stopFloating();
+            this.stopFloating();
         }
     }
 
-    function _stopFloating(forever) {
+    this.stopFloating = function(forever) {
```

Comment:
Does this method still need to be on the instance?

Reply:
Yes, it is being called in commercial

Reply:
Ok I see, thanks